src/components/TodoItem/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './TodoItem';
```

Comment:
You should import the module first and then export it separately, as per the given rules. 📝